apps/extension/src/webview/index.tsx

```
@@ -0,0 +1,1709 @@
+			const updatedTasks = state.tasks.map((task) =>
+				task.id === action.payload.taskId
+					? { ...task, status: action.payload.newStatus }
+					: task
+			);
+			return { ...state, tasks: updatedTasks };
+		case 'UPDATE_TASK_CONTENT':
+			const updatedTasksContent = state.tasks.map((task) =>
+				task.id === action.payload.taskId
+					? { ...task, ...action.payload.updates }
+					: task
+			);
```

Biome (1.9.4)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)